proposals/2019/Move Plants from HVAC Systems to Facilities.md

```diff
@@ -0,0 +1,58 @@
+# Move Plants from HVAC Systems to Facilities
+
+## Overview
+
+This proposal is to move the `<auc:Plants>` element from the `<auc:HVACSystemType>` element to the `<auc:Facility>` element.
+
+## Justification
+
+The justification is three-fold:
+
+1. Plants cannot be asserted without first asserting the HVAC system, i.e., the `<auc:Plants>` element can only be included in an XML document if it is a child element of an `<auc:HVACSystemType>` element.
+
+2. If there is more than one HVAC system in a given facility and a plant is shared between two or more HVAC systems, then there is no rule for where the plant should be asserted, i.e., the `<auc:Plants>` element can be included as the child element of any `<auc:HVACSystemType>` element within a given XML document.
```

Comment:
@jasondegraw The current definition of the XML schema assumes that an HVAC system aggregates zero or more plants. However, the definition of the Audit Template database schema assumes that a facility/building aggregates zero more more HVAC systems and zero or more plants. 

Suppose that you have a facility with two HVAC systems and one plant. The XML schema is ambiguous because the user can generate two XML documents with equivalent interpretations: the first with the plant "in" the first HVAC system; and the second with the plant "in" the second HVAC system.

Reply:
@markborkum The plants here _are_  HVAC systems, so your example has three HVAC systems, not two. So again, there are options, but that doesn't imply ambiguity, and it certainly doesn't justify the proposed change.